GitUI/Script/ScriptRunner.cs

```diff
@@ -57,7 +57,7 @@ public static CommandStatus RunScript(IWin32Window owner, GitModule module, stri
                     continue;
                 }
 
-                if (!option.StartsWith("{s"))
+                if (!option.StartsWith("s"))
```

Comment:
Is there a test we could add?

Reply:
see my short comment https://github.com/gitextensions/gitextensions/pull/6502#issuecomment-485584736 above and https://github.com/gitextensions/gitextensions/pull/6491#issuecomment-485569990